nengo/builder/connection.py

```diff
@@ -269,11 +275,16 @@ def get_prepost_signal(is_pre):
         weights, name="%s.weights" % conn, readonly=True)
     signal = Signal(np.zeros(signal_size), name="%s.weighted" % conn)
     model.add_op(Reset(signal))
-    op = ElementwiseInc if weights.ndim < 2 else DotInc
-    model.add_op(op(model.sig[conn]['weights'],
-                    in_signal,
-                    signal,
-                    tag="%s.weights_elementwiseinc" % conn))
+
+    if isinstance(conn.transform, Convolution):
```

Comment:
The problem with this is if somebody wants to add their own transform, they'd have to list it here, which is not really possible. Would it be possible to have a `build` function for the transform, to make it pluggable?